### PR TITLE
Release v2.2.4

### PR DESCRIPTION
**Pull request by HanlinMiao**
- [x] towncrier changelogs
- [x] Version bump
- [x] Compatibility Matrix

### CODE REVIEW

File: docs/admin/release_notes/version_2.2.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+
+### Fixed
+
+- [#312](https://github.com/nautobot/nautobot-app-circuit-maintenance/issues/312) - Fixed incorrect reference to nonexistent `raw_notification__date` field that caused errors under Django 4.2.
+
+### Documentation
+
+- [#345](https://github.com/nautobot/nautobot-app-circuit-maintenance/issues/345) - Updated app config and urls to populate documentation link.
+
 ## [v2.2.2 (2024-06-04)](https://github.com/nautobot/nautobot-app-circuit-maintenance/releases/tag/v2.2.2)
```
Comment (glennmatthews) [inline docs/admin/release_notes/version_2.2.md:15]: No release-note content for v2.2.3?

Comment (glennmatthews) [reply docs/admin/release_notes/version_2.2.md:15]: ah, looks like #345 was in 2.2.3.

File: docs/admin/release_notes/version_2.2.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+## [v2.2.4 (2024-08-07)](https://github.com/nautobot/nautobot-app-circuit-maintenance/releases/tag/v2.2.4)
+
+### Fixed
+
+- [#312](https://github.com/nautobot/nautobot-app-circuit-maintenance/issues/312) - Fixed incorrect reference to nonexistent `raw_notification__date` field that caused errors under Django 4.2.
+
+### Documentation
+
+- [#345](https://github.com/nautobot/nautobot-app-circuit-maintenance/issues/345) - Updated app config and urls to populate documentation link.
+
```
Comment (glennmatthews) [inline docs/admin/release_notes/version_2.2.md:14]:
```suggestion
## [v2.2.4 (2024-08-07)](https://github.com/nautobot/nautobot-app-circuit-maintenance/releases/tag/v2.2.4)

### Fixed

- [#312](https://github.com/nautobot/nautobot-app-circuit-maintenance/issues/312) - Fixed incorrect reference to nonexistent `raw_notification__date` field that caused errors under Django 4.2.

## [v2.2.3 (2024-07-03)](https://github.com/nautobot/nautobot-app-circuit-maintenance/releases/tag/v2.2.3)

### Documentation

- [#345](https://github.com/nautobot/nautobot-app-circuit-maintenance/issues/345) - Updated app config and urls to populate documentation link.

```

**Review by glennmatthews — APPROVED** [review]

File: docs/admin/release_notes/version_2.2.md (hunk 1 above)
Comment (HanlinMiao) [reply docs/admin/release_notes/version_2.2.md:15]: I just need to merge main into my branch here.

**Review by glennmatthews — APPROVED** [review]

**Comment by gsnider2195** [comment]
We need to update the compatibility matrix to show that this is the first version to support Nautobot v2.3.0

File: docs/admin/compatibility_matrix.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 | 0.1.1-0.6.2   | 1.0.0                | 1.99.99       |
 | 1.0.0         | 1.4.0                | 1.5.99        |
 | 2.0.0         | 2.0.0                | 2.99.99       |
+| 2.2.4         | 2.3.0                | 2.99.99       |
```
Comment (glennmatthews) [inline docs/admin/compatibility_matrix.md:8]: Presumably:
```suggestion
| 2.0.0-2.2.3   | 2.0.0                | 2.2.99       |
| 2.2.4         | 2.0.0                | 2.99.99       |
```

**Review by gsnider2195 — APPROVED** [review]

File: docs/admin/release_notes/version_2.2.md [hunk 4]
```diff
@@ -2,13 +2,18 @@
 
 <!-- towncrier release notes start -->
 
+## [v2.2.4 (2024-08-07)](https://github.com/nautobot/nautobot-app-circuit-maintenance/releases/tag/v2.2.4)
```
Comment (gsnider2195) [inline docs/admin/release_notes/version_2.2.md:5]:
```suggestion
## [v2.2.4 (2024-08-08)](https://github.com/nautobot/nautobot-app-circuit-maintenance/releases/tag/v2.2.4)
```

**Pull request merged**